src/NetEscapades.EnumGenerators/SourceGenerationHelper.cs

```diff
@@ -1433,4 +1434,15 @@ public static string[] GetNames()
             .ToString();
         return (content, filename);
     }
+
+    private static IReadOnlyCollection<(string Key, EnumValueOption Value)> GetNamesOrderedByValue(EnumToGenerate enumToGenerate)
```

Comment:
nit: we can just return `List<>` here, no need for the interface
```suggestion
    private static List<(string Key, EnumValueOption Value)> GetNamesOrderedByValue(EnumToGenerate enumToGenerate)
```

Reply:
Yeah, that was a reminder of the previous implementation, were an `EquatableArray` could have been returned, if the values were already in order